src/components/Filter/OptionDisplay.module.css

```diff
@@ -9,16 +9,16 @@
   display: flex;
   align-items: start;
   flex-direction: column;
-  border-left: 1px solid var(--fds-semantic-border-divider-default);
-  border-right: 1px solid var(--fds-semantic-border-divider-default);
-  border-top: 1px solid var(--fds-semantic-border-divider-default);
+  border-left: 1px solid var(--ds-color-neutral-contrast-subtle);
+  border-right: 1px solid var(--ds-color-neutral-contrast-subtle);
+  border-top: 1px solid var(--ds-color-neutral-contrast-subtle);
 }
 
 .option {
   width: 100%;
   border: none;
-  border-bottom: 1px solid var(--fds-semantic-border-divider-default);
-  background-color: var(--fds-semantic-background-default);
+  border-bottom: 1px solid var(--ds-color-neutral-contrast-subtle);
+  background-color: var(--ds-color-neutral-contrast-default);
   cursor: pointer;
   font: inherit;
   text-align: left;
```

Comment:
For the line under:
The options of the filter look a bit smaller than before for some reason.
I suggest changing to `padding: 8px 0;`

Reply:
👌